common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -531,13 +533,37 @@ String sign(@NonNull final SigningAlgorithm alg,
         final Exception exception;
         final String errCode;
 
+        final byte[] inputBytesToSign;
+        try {
+            inputBytesToSign = input.getBytes(ENCODING_UTF8);
+            return Base64.encodeToString(sign(alg, inputBytesToSign), Base64.NO_WRAP);
+        }  catch (final UnsupportedEncodingException e) {
+            final ClientException clientException = new ClientException(
+                    UNSUPPORTED_ENCODING,
+                    e.getMessage(),
+                    e
+            );
+
+            Logger.error(
+                    TAG + methodName,
+                    clientException.getMessage(),
+                    clientException
+            );
+            throw clientException;
+        }
+
+    }
+
+    @Override
+    public byte[] sign(@NonNull SigningAlgorithm alg, byte[] inputBytesToSign) throws ClientException {
+        Exception exception;
+        String errCode;
         try {
-            final byte[] inputBytesToSign = input.getBytes(ENCODING_UTF8);
             final KeyStore.Entry keyEntry = mKeyStore.getEntry(mKeyAlias, null);
 
             if (!(keyEntry instanceof KeyStore.PrivateKeyEntry)) {
                 Logger.warn(
-                        TAG + methodName,
+                        TAG + ":sign",
```

Comment:
Perhaps we can use `final String methodName = ":sign"` instead of having to repeat the `":sign"`.

Reply:
done